jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile

```diff
@@ -132,7 +132,7 @@ pipeline {
             steps {
                 gitCommitWhenChanges("Updated image_mirror_mapping") { msgs ->
                     echo "Process Manifest Images - generate image_mirror_mapping files and update CSV with new images tags"
-                    sh "scripts/process-csv-images ${productName}"
+                    sh "scripts/ews/process-csv-images ${productName}"
```

Comment:
Do we think ews is an ok name here or do we want to more explicit, delorean, delorea-ews? Note, This will be updated in the delorean-ews operator branch before i merge this.

Reply:
@laurafitzgerald @StevenTobin ?

Reply:
Related https://github.com/integr8ly/integreatly-operator/pull/493

Reply:
I think delorean or delorean-ews would be better. Not everyone knows what ews means.

Reply:
Yep, that's what i was thinking, lets go with delorean for now. Will update.